astroplan/plots/sky.py

```diff
@@ -0,0 +1,195 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import matplotlib.pyplot as plt
+import numpy as np
+import astropy.units as u
+
+
+@u.quantity_input(az_label_offset=u.deg)
+def plot_sky(target, observer, time, ax=None, style_kwargs=None,
+             north_to_east_ccw=True, grid=True, az_label_offset=0.0*u.deg):
+    """
+    Plots target positions in the sky with respect to the observer's location.
+
+    If an ax object already exists, plots an additional target position mapped
+    on top.
+    Otherwise, creates a new ax object with a sky plot.
+
+    Can pass in a scalar `Time` object (e.g., Time('2000-1-1')) or a non-scalar
+    one (e.g., Time(['2000-1-1'])).
+    If pass in `Time` objects with multiple instances of time
+    (e.g., Time(['2000-1-1 20:00:00', '2000-1-1 20:30:00'])), target's position
+    will be shown at each of these times.
+
+    Parameters
+    ----------
+    target: `~astroplan.FixedTarget`
+        The celestial body of interest.
+
+    observer: `~astroplan.Observer`
+        The person, telescope, observatory, etc. doing the observing.
+
+    time : `~astropy.time.Time`
+        Can be scalar (e.g., Time('2000-1-1')) or not
+        (e.g., Time(['2000-1-1'])).
+
+    ax : `~matplotlib.axes.Axes` or None, optional.
+        The axes object to be drawn on.
+        If None, use the current axes (`matplotlib.pyplot.gca`).
+
+    style_kwargs : dict or Empty, optional.
+        A dictionary of keywords passed into `matplotlib.pyplot.plot_date`
```

Comment:
I think this is a typo in that it is passed into `scatter`, not `plot_date`, right?